src/python/TaskWorker/Actions/DagmanCreator.py

```diff
@@ -215,6 +215,28 @@ def makeLFNPrefixes(task):
 
     return temp_dest, dest
 
+def validateLFNs(path, files):
+    """
+    validate against standard Lexicon the LFN's that this task will try to publish in DBS
+    :param path: string: the path as created by CRAB starting from task name and possibly user LNFbase
+    :param files: list of strings: the filenames that will be published
+    :return: nothing if all OK. Raise assert exception if failure (as Lexicon functions do)
+    This is meant to be used early in the task preparation,
+    when the path and files passed do not have yet the directory counter and job_id counter,
+    so will insert take ones here when doing the check to ensure LFN lenght is correct
+    """
+    from WMCore import Lexicon
+    count = 10000     # at most 10k jobs in a task, 4 chars must suffice
+    counter = '0001'  # 4 digits for the directory counter
```

Comment:
Can you rename these `count, counter` variables to something more meaningful? Current names give the impression that they're being used in the logic but they are just placeholders. Also, they should both be strings IMO.